### ISSUE TITLE
Pose and Scale in dataset

### ISSUE THREAD

**Issue by SuperStacie** [YuelangX/LatentAvatar#4]
Hi, thanks for the interesting work!

I wonder if the pose and scale in the dataset are being used during the training and inference. And how do we prepare the data 
 acquire the camera parameters if we wanna try it out using self-captured data? 

Thanks

**Comment by iperov** [comment]
Yeah, they didn't specify in the paper that the dataset requires the calculation of pose and some sort of scale for each frame that go into training in Nerf.
It looks like these values are calculated somehow from landmarks.
Given that their examples have shaky frames, it looks like pose and scale play a key role ?

**Comment by iperov** [comment]
but according https://github.com/YuelangX/LatentAvatar/blob/main/lib/module/HeadModule.py
they use pose+scale only color_mlp in nerf, but not in density_mlp. Perhaps we don't need that embedding?

**Comment by wuzuyin** [comment]
How long does it take to train approximately

**Comment by wuzuyin** [comment]
> Hi, thanks for the interesting work!
> 
> I wonder if the pose and scale in the dataset are being used during the training and inference. And how do we prepare the data acquire the camera parameters if we wanna try it out using self-captured data?
> 
> Thanks

Hello, have you resolved this issue？

**Comment by wuzuyin** [comment]
> +1
Hello, have you resolved this issue？
